app/routes/screens/race/Race.js

```diff
@@ -52,7 +53,8 @@ class RacePredictor extends Component {
           })
         })
       )
-    }).isRequired
+    }).isRequired,
+    geoloactionActionsCreators: PropTypes.objectOf(PropTypes.func).isRequired
   };
 
   static intToColor(id = 0) {
```

Comment:
Right now this was only useful in that context.  That is why it was just a static function of that class.